apps/frontend/src/pages/no-access/no-access.tsx

```diff
@@ -0,0 +1,19 @@
+import { PageLayout } from "~/libs/components/components.js";
+
+import styles from "./styles.module.css";
+
+const NoAccess = (): JSX.Element => {
+	return (
+		<PageLayout>
+			<div className={styles["container"]}>
+				<h1 className={styles["title"]}>Pending access</h1>
+				<p className={styles["text"]}>
+					You do not have access to any pages for now. Please contact an admin
```

Comment:
I don't see any styles added that will restrict the text to have fixed container and not going full page width 🤔

Reply:
Do I have to add something like `max-width: 800px;`?

Reply:
yes, I think 800 is too much, 400-450 should be enough